src/snowflake/snowpark/_internal/proto/update-from-devvm.sh

```diff
@@ -1,19 +1,21 @@
 #!/usr/bin/env bash
-# This is a little helper script to build on the devvm the python protobuf file and copy it over to this directory.
-# Run script from within this directory.
-# For this script to succeed, need to configure SSH devvm host according to
+# This is a little helper script to build the python protobuf file and copy it over to this directory.
+# Run script from within this directory, and provide the Host as the argument.
+# Cloud workspace ssh is set up via ~/.local/share/sfcli/ssh_config (or simply provide your workspace ID)
+# For this script to succeed with the DevVM, you need to configure the SSH devvm host according to
 # https://snowflakecomputing.atlassian.net/wiki/spaces/EN/pages/3019015180/DevVM+Troubleshooting#Symptom:-Need-to-ssh-into-the-certified-DevVM
 # I.e., add the following lines to ~/.ssh/config
 # Host devvm
 #	  HostName sdp-devvm-<ldap user>
 
 set -euxo pipefail
+SCRIPT_DIR=$(dirname "$0")
 
-# Step 1: Built python proto file from scratch via bazel
-ssh devvm "bash -c 'cd Snowflake/trunk;bazel build //Snowpark:py_proto'"
+# Step 1: Build the python proto file from scratch via bazel
+ssh $1 "bash -c 'cd Snowflake/trunk;bazel build //Snowpark:py_proto'"
```

Comment:
maybe some checks on `$1`? we could also make this optional, i.e. specify a variable `HOST` and set it to $1 if available, print help if not given etc.